sdks/python/apache_beam/testing/test_stream_test.py

```diff
@@ -195,15 +195,12 @@ def test_gbk_execution_after_watermark_trigger(self):
 
     # assert per window
     expected_window_to_elements = {
-        window.IntervalWindow(15, 30): [
-            ('k', ['a']),
-            ('k', []),
-        ],
+        window.IntervalWindow(0, 15): [('k', ['a']), ('k', [])],
```

Comment:
The window changed because the test was silently failing. The element was being emitted from the [0, 15) window, not the [15, 30) window. The fixed matcher caught the bug.